app/views/content/life-as-a-teacher/explore-subjects/languages.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Become a languages teacher
+title_paragraph: |-
+  <p>
+  Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>
+  <p>
+  Bursaries of $bursaries_postgraduate_languagesother$ are available for all trainee language teachers. These bursaries include ancient languages.</p> 
```

Comment:
We need to dial this back for now as per the holding message PR - we can't say amounts without a caveat so @emmaachesongray has made the other subject pages very generic in this header section - need to do the same here!